source/solvers/mf_navier_stokes.cc

```diff
@@ -452,13 +452,6 @@ MFNavierStokesPreconditionGMG<dim>::initialize_ls(
       amg_data.output_details = false;
       amg_data.smoother_type  = "ILU";
       amg_data.coarse_type    = "ILU";
-      // Constant modes for velocity and pressure
-      // std::vector<std::vector<bool>> constant_modes;
-      // ComponentMask                  components(dim + 1, true);
-      // DoFTools::extract_constant_modes(dof_handler,
-      //                                  components,
-      //                                  constant_modes);
-      // amg_data.constant_modes = constant_modes;
```

Comment:
We should keep this as a reminder that we need to enable this for local smoothing.

Reply:
Agreed